#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;


gen = ParameterGenerator ()
gen.add("keep_organized", bool_t, 0, "keep organized pointcloud or not", True)
gen.add("mean_k", int_t, 0, "The number of points (k) to use for mean distance estimation", 2, 2, 100)
gen.add("stddev", double_t, 0, "The standard deviation multiplier threshold. All points outside the mean +- sigma * std_mul will be considered outliers.", 0.0, 0.0, 5.0)
gen.add("negative", bool_t, 0, "Set whether the inliers should be returned (true) or the outliers (false)", False)

exit(gen.generate (PACKAGE, "jsk_pcl_ros", "OrganizedStatisticalOutlierRemoval"))
